src/main/java/seedu/address/logic/commands/ScienceCommand.java

```diff
@@ -51,10 +51,14 @@ public CommandResult execute(Model model) {
             setScienceModules(SCIENCE_PATH);
             StringBuilder modulesToAdd = new StringBuilder();
             for (Module module : scienceModules) {
-                String moduleToAdd = module.getModuleCode() + " (" + module.getModularCredits() + " MCs)";
+                if (model.hasModule(module)) {
+                    continue;
+                }
+                String moduleToAdd = module.getModuleCode() + "\t" + module.getModuleTitle()
+                    + " (" + module.getModularCredits() + " MCs)";
                 modulesToAdd.append("\n").append(moduleToAdd);
```

Comment:
Really damn small nit but maybe the happy path could be made clearer since it's actually just a single path: 
```suggestion
                if (!model.hasModule(module)) {
                    String moduleToAdd = module.getModuleCode() + "\t" + module.getModuleTitle()
                        + " (" + module.getModularCredits() + " MCs)";
                    modulesToAdd.append("\n").append(moduleToAdd);
                }
```